indispensable/src/jvmTest/kotlin/at/asitplus/signum/indispensable/OidTest.kt

```diff
@@ -27,13 +31,17 @@ class OidTest : FreeSpec({
 
         "Automated" - {
             checkAll(iterations = 15, Arb.positiveInt(39)) { second ->
-                checkAll(iterations = 1000, Arb.intArray(Arb.int(0..128), Arb.positiveInt(Int.MAX_VALUE))) {
+                checkAll(iterations = 5000, Arb.intArray(Arb.int(0..128), Arb.positiveInt(Int.MAX_VALUE))) {
```

Comment:
Really necessary to make it 5000 instead of 1000?

Reply:
it is really, really fast now, so it won't matter